resources/translations/en.edn

```diff
@@ -61,6 +64,7 @@
             :decide "Decide"
             :description "Description"
             :empty "You don't have any applications to process."
+            :errors {:invalid-token [:div [:p "Joining the application failed."] [:p "Check the invitation token " [:code "%1"]]]}
```

Comment:
can the user really check the code somehow?

Reply:
The code will be shown in the mail with the link to join. Some people like to type links by hand in the address bar. There's not much else that can be said?

Reply:
ok fair enough